.github/workflows/report-vulnerability-in-gh.yaml

```diff
@@ -0,0 +1,78 @@
+name: Report vulnerability issues in Github
+on:
+  workflow_call:
+    inputs:
+      issue-title:
+        description: The title of the issue to be created/edited
+        required: true
+        type: string
+      issue-labels:
+        description: A comma separated list of labels
+        required: false
+        type: string
+        default: "bug"
+      image-name:
+        description: "Name of the oci-image as saved in Dockerhub or in the docker cache.
+          It consists of <rock-name>:<rock-version>."
+        required: true
+        type: string
+
+jobs:
+  report-vulns:
+    name: Create or edit issues for reporting vulnerabilities
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install tools
+        run: |
+          sudo snap install gh
+          sudo snap install jq
+
+      - name: Generate image name
+        id: image-name
+        run: |
+          IMAGE_NAME=$(echo ${{ inputs.image-name }} | sed 's/\:/-/g')
```

Comment:
Do we need to do this?

Reply:
Actually, I think I might've made a mistake and forgot to remove this, as in a workflow where I'm calling this re-usable workflow, I pass the image name with dashes `image-name: ${{ needs.scan.outputs.image-name-dashes }}` from #73.

I think in that other PR you are suggesting to pass the image name in the `<image>:<tag>` format, so I think it makes more sense to:

* take as input `image-name` with the format of `<image>:<tag>`
* take as input `vulnerability-report-artefact`

I'll make the change to make this more clear.

Reply:
+1, it indeed makes the API more clear